src/adv.py

```diff
@@ -49,3 +56,31 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+# input .strip() - avoids having error when the user's input has empty spaces or tabs
+
+# Create a player 
+# Let player input their name
+
+player = Player(input("Please enter your name: "), room['outside'])
+print(player.current_room)
+
+print(player.name)
+
+print(room['outside'].n_to)
+
+directions = ["n", "s", "e", "w"]
+
+# Create basic REPL loop 
+while True:
+    # Read command 
+    cmd = input("~~> ").lower()
+    # Check if it's n/s/e/w/q
+    if cmd in directions:
+        # Make player travel in that direction
+        player.travel(cmd)
+    elif cmd == "q":
+        print("Goodbye!")
+        exit()
+    else:
```

Comment:
Using your else block is great, and you should almost always add it, because even if you've written a conditional for every case you can imagine, you never know what users will do when your program is in the wild! For instance they could type in a symbol, and if you didn't have this else block, the program may crash!

Reply:
I will keep that in mind. Thanks!